script.js

```diff
@@ -0,0 +1,369 @@
+
+var counter = 0;
+var state1 = true;
+var state2=true;
+var state3=true;
+var state4=true;
+var state5=true;
+var state6=true;
+var state7=true;
+var state8=true;
+var state9=true;
+
+function myFunction1() {
+
+
+    if(counter % 2==0 && state1 ==true)
+    {
+
+    var add = (function () {
+      document.getElementById("cell1").style.color = "Blue";
+      document.getElementById("cell1").innerHTML = "0";
+      state1 =false;
+      return function () {return counter += 1; }
+    })();
+
+    add();
+
+    }
+    else if(state1 ==true){
+
+
+      var add = (function () {
+        document.getElementById("cell1").innerHTML = "X";
+state1 =false;
+        return function () {return counter += 1;}
+      })();
+
+      add();
+
+}
+
+else{ }
+
+
+}
+
+
+function myFunction2() {
+
```

Comment:
please remove white space and indent code correctly before submitting in the future